notebooks/00-getting_started.ipynb

```diff
@@ -194,7 +172,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "prediction = axon_segmentation([path_testing], [\"image.png\"], path_model, config_network, resampled_resolutions=0.1, verbosity_level=3)"
+    "segment_image(\n",
+    "        path_testing_image=path_testing / 'image.png',\n",
+    "        path_model=path_model,\n",
+    "        overlap_value=[25, 25],\n",
+    "        acquired_resolution=0.37\n",
```

Comment:
```suggestion
    "        acquired_resolution=0.13\n",
```
Hum, I don't remember a pixel size of 0.37 anywhere. In this case, it is a large resampling and increases significantly the number of patches and the segmentation time (+ the result is very bad). 0.13 is the real pixel size of the test image.

Reply:
Resolved in 2dbc5f768